packages/mask/src/plugins/RedPacket/SNSAdaptor/utils/chain.ts

```diff
@@ -105,52 +102,5 @@ export async function getRedPacketHistory(
         }
     })
     // #endregion
-
-    // #region
-    // 3. Decode CreationSuccess event log to retrieve `rpid` and `creation_time` for payload
-    type CreationSuccessEventParams = {
-        id: string
-        creation_time: BigNumber
-    }
-
-    const eventLogResponse = await Promise.allSettled(
-        payloadList.map(async (payload) => {
-            const result = await connection.getTransactionReceipt(payload.txid)
-            if (!result) return null
-
-            const log = result.logs.find((log) => isSameAddress(log.address, HAPPY_RED_PACKET_ADDRESS_V4))
-            if (!log) return null
-            const eventParams = interFace.decodeEventLog(
-                'CreationSuccess',
-                log.data,
-                log.topics,
-            ) as unknown as CreationSuccessEventParams
-
-            payload.rpid = eventParams.id
-            payload.creation_time = eventParams.creation_time.toNumber() * 1000
-
-            // 4. retrieve `claimers` and `total_remaining`
-            // const r = await redPacketContract.methods.check_availability(payload.rpid).call({
-            //     from: payload.sender.address,
-            // })
-
-            const data = await checkAvailability(
```

Comment:
Remove the duplicated `checkAvailability()` here since `<RedPacketInHistoryList />` uses `useAvailabilityComputed()`.